schemerz-postgres/CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this library adheres to Rust's notion of
 <!-- next-header -->
 ## [Unreleased]
 
+### Changed
+- **IMPORTANT BREAKING CHANGE**: `schemerz_postgres::PostgresAdapter::new` now
+  uses a default table name of `_schemerz` when the `table_name` argument is
+  `None`. If you were not setting this argument before and are migrating from
+  `schemer`, you will need to set `table_name` to `Some("_schemer".into())`.
```

Comment:
We were the only public dependent on `schemer`, so I think this notice should be all we need to do. But an alternative would be to remove the default and require the caller to always specify the table name.